docs/Configuration-Settings.md

```diff
@@ -448,6 +448,21 @@ The user configuration specifies values regarding file system access.
   <td>3000</td>
   <td>The maximum number of milliseconds for a netty client (for block reads and block writes) to wait for a response from the data server.</td>
 </tr>
+<tr>
+  <td>tachyon.user.remote.block.worker.threads</td>
+  <td>10</td>
+  <td>How many threads to use for remote block worker.</td>
```

Comment:
it is still the number threads in the pool waiting for clients talking to a (arbitrary) remote block worker. So maybe name the property name consistent ?